blackbox/stratisd.spec

```diff
@@ -40,9 +40,8 @@ Stratisd test build.  This package should not be used in production
 
 
 %prep
-%setup -q -n %{name}-%{version}
-%patch0 -p1
-
+%autosetup -p1
```

Comment:
Better off going back to ```%setup -q``` here. With this setup, we will have to apply any patches _after_ untarring the Sourc2 file, not before.

Reply:
Resolved.